apps/codeforafrica/src/components/ArticlePage/ArticlePage.js

```diff
@@ -80,18 +79,14 @@ function ArticlePage({
           px: { xs: 2.5, sm: 0 },
         }}
       >
-        <Author {...author} />
+        {/* 
+        //TODO: confirm if this is to be shown on opportunities page
+         */}
+
+        <Author {...(primary_author || authors[0])} />
```

Comment:
@kilemensi Author should only be shown only on a story page, and not on the opportunity page right?

Reply:
I kinda of agree @kelvinkipruto but [designs](https://www.figma.com/file/MbCygAaNhIjgJOHrQZpZ2U/CfA---CfA-site?node-id=7255%3A39753) says we should show author in both.

Reply:
1. There is a feedback review period after development where we can raise these issues. Since we do have the value, lets show it for now.
2. I believe there is `bio` field for authors. We can use that for now, no?

Reply:
Ok. I agree. Lets use `bio` for now.